internal/cephfs/controllerserver.go

```diff
@@ -151,6 +154,7 @@ func checkContentSource(
 	req *csi.CreateVolumeRequest,
 	cr *util.Credentials,
 	clusterName string,
+	setMetadata bool,
```

Comment:
you can make this `checkContentSource()` function a method of `ControllerServer`. That way there is no need to pass the new two arguments `clusterName` and `setMetadata`. They can then be used directly as `cs.ClusterName` and `cs.SetMetadata`.